standard/expressions.md

```diff
@@ -5715,6 +5785,7 @@ A generic type `C<T>` supports the query-expression-pattern if its public member
 
 <!-- IncompleteExample: {template:"standalone-lib", name:"Query-expressionPattern", expectedErrors:["x","x"], expectedWarnings:["x","x"]} -->
 ```csharp
+delegate R Func<T1,R>(T1 arg1);
```

Comment:
Do we need either of these declarations now, if we've got using directives for System?

Reply:
I'll review this in the next round when I try to complete this example.